x-pack/plugins/observability_solution/inventory/public/components/entities_grid/entity_name/index.tsx

```diff
@@ -42,47 +33,25 @@ export function EntityName({ entity }: EntityNameProps) {
     });
   }, [entity, telemetry]);
 
-  const getEntityRedirectUrl = useCallback(() => {
-    const type = entity[ENTITY_TYPE];
-    // For service, host and container type there is only one identity field
-    const identityField = Array.isArray(entity[ENTITY_IDENTITY_FIELDS])
-      ? entity[ENTITY_IDENTITY_FIELDS][0]
-      : entity[ENTITY_IDENTITY_FIELDS];
-    const identityValue = entity[identityField];
-
-    switch (type) {
-      case 'host':
-      case 'container':
-        return assetDetailsLocator?.getRedirectUrl({
-          assetId: identityValue,
-          assetType: type,
-        });
-
-      case 'service':
-        return serviceOverviewLocator?.getRedirectUrl({
-          serviceName: identityValue,
-          environment: [entity[SERVICE_ENVIRONMENT] || undefined].flat()[0],
-        });
-    }
-  }, [entity, assetDetailsLocator, serviceOverviewLocator]);
+  const entityName = (
+    <EuiFlexGroup gutterSize="s" alignItems="center">
+      <EuiFlexItem grow={0}>
+        <EntityIcon entity={entity} />
+      </EuiFlexItem>
+      <EuiFlexItem className="eui-textTruncate">
+        <span className="eui-textTruncate" data-test-subj="entityNameDisplayName">
+          {entity[ENTITY_DISPLAY_NAME]}
```

Comment:
Can you do?
```suggestion
          {entity.display_name}
```

Reply:
Currently, no. 

I'm finalizing this [#198758](https://github.com/elastic/kibana/issues/198758) and will open a PR as soon as this one is merged.

Reply:
Ah you created the new type but haven't replaced the old one, right? 

https://github.com/elastic/kibana/pull/197222/files#diff-b0d4049f68110aa54f26b9a5eae6026ab496059109de95a1ef27b686651ac1ccR122

Reply:
Not yet. I created that type to facilitate the call to Entity Client's helper functions. I'll replace the old type on the other PR.